scan_test.py

```diff
@@ -246,6 +246,10 @@ def test_sns_start_scan(self):
             "TargetArn": sns_arn,
             "Message": json.dumps({"default": json.dumps(message)}),
             "MessageStructure": "json",
+            "MessageAttributes": {
+                "bucket": {"DataType": "String", "StringValue": self.s3_bucket_name},
+                "key": {"DataType": "String", "StringValue": self.s3_key_name},
```

Comment:
Is it possible to mock `bucket_name` and `key_name` and use literals here?

Reply:
I didn't want to go against the code style of that repo. `bucket_name` and `key_name` are test variables defined in the setUp method before each test https://github.com/jobready/bucket-antivirus-function/blob/master/scan_test.py#L43-L44 and used ubiquitously throughout the tests. There is no possibility of the application code mutating them.